src/sentry/static/sentry/app/stores/hookStore.tsx

```diff
@@ -4,13 +4,10 @@ import * as Sentry from '@sentry/browser';
 
 import {Hooks, HookName} from 'app/types/hooks';
 
-type HookMapping = {[H in HookName]?: Array<Hooks[H]>};
-
 /**
  * See types/hooks for hook usage reference.
  */
 const validHookNames = new Set<HookName>([
-  '_',
```

Comment:
What was this?

Reply:
Updated to remove the comment that explains it

Reply:
nevermind, still need this hack.